tests/integration/components/popup-test.js

```diff
@@ -142,3 +142,16 @@ test('popup closes when layer is destroyed', function(assert) {
   assert.equal(map._popup, null, 'popup closed');
 
 });
+
+test('popupOptions hash', function(assert) {
+  this.set('markerCenter', locations.nyc);
+  this.render(hbs`
+    {{#leaflet-map zoom=zoom center=center}}
+      {{#marker-layer location=markerCenter draggable=draggable popupOptions=(hash className="foo")}}
```

Comment:
It's probably better to use `popupOptions=popupOptions`
and then
```js
this.set('popupOptions', { className: 'foo' });
```

this way the tests pass on all ember versions without the `hash` polyfill.